src/js/formprint/index.js

```diff
@@ -0,0 +1,98 @@
+import 'js/base/setup';
+import 'js/i18n';
+
+import Backbone from 'backbone';
+import Radio from 'backbone.radio';
+import { View, CollectionView } from 'marionette';
+import hbs from 'handlebars-inline-precompile';
+
+import 'sass/provider-core.scss';
+import 'sass/app-root.scss';
+
+import './formprint.scss';
+
+import initPlatform from 'js/utils/platform';
+
+import App from 'js/base/app';
+
+import FormsService from 'js/services/forms';
+
+import 'js/entities-service';
+
+import BootstrapService from 'js/services/bootstrap';
+
+import IframeFormBehavior from 'js/behaviors/iframe-form';
+
+const RootView = CollectionView.extend({
+  viewComparator: false,
+  el: 'body',
+  template: hbs``,
+  initialize({ model }) {
+    this.render();
+
+    this.addChildView(new FormIframeView({ model }));
+  },
+});
+
+const FormIframeView = View.extend({
+  behaviors: [IframeFormBehavior],
+  className: 'form__frame',
+  template: hbs`
+    <div class="app-frame__content flex-region">
+      <div class="form__layout">
+        <div class="form__content">
+          <iframe src="/formapp/"></iframe>
+        </div>
+      </div>
+    </div>
+  `,
+});
+
+const FormPrintApp = App.extend({
+  channelName: 'app',
+
+  initialize() {
+    initPlatform();
```

Comment:
I think we won't need this

Reply:
Yup, removed this without issue. Good call.